cmd/root.go

```diff
@@ -11,11 +11,11 @@ import (
 	"github.com/spf13/cobra"
 )
 
-const defaultLogLevel = zerolog.ErrorLevel
+const defaultLogLevel = zerolog.InfoLevel
 
 // rootCmd represents the base command when called without any subcommands
 var rootCmd = createRootCommand()
-var logger zerolog.Logger
+var logger = log.With().Str("component", "cmd").Logger()
```

Comment:
Shouldn't this be using the logger package somehow?

Reply:
The logger package is only used for initialization of zerolog via side-effect `log.With()...` creates a copy of `log.Logger`, the global logger, which was configured by the logger package.

Reply:
Thanks, lots track of it in the end :)